pkg/cluster/internal/create/actions/kubeadmjoin/join.go

```diff
@@ -117,18 +118,31 @@ func joinWorkers(
 
 // runKubeadmJoin executes kubeadm join command
 func runKubeadmJoin(logger log.Logger, node nodes.Node) error {
-	// run kubeadm join
-	// TODO(bentheelder): this should be using the config file
```

Comment:
@BenTheElder It seemed like this was the last missing piece for this TODO. Please let me know if there's still a gap here where this comment is still valuable.

Reply:
yep! thanks for fixing my ancient TODO 😅